app/models/site_config.rb

```diff
@@ -206,6 +206,9 @@ class SiteConfig < RailsSettings::Base
     xlarge: 250
   }
 
+  # Push Notifications
+  field :push_notifications_ios_pem, type: :string
```

Comment:
The future of this could lead to us more towards a `MobileApp` model rather than a `SiteConfig`. In that case we would store the keys (like this one) for any Mobile App and each Forem instance could send PNs to N number of apps (managed by the creator admin dashboard)